PORTING-ADDONS-TO-V2.md

```diff
@@ -74,6 +74,10 @@ The steps:
    # you definitely want this:
    node_modules
 
+   # you don't want *.md copies
+   /addon/README.md
+   /addon/LICENSE.md
```

Comment:
Would you mind sending a PR to the blueprint for these also? https://github.com/embroider-build/addon-blueprint/blob/main/files/gitignore